compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/Types.java

```diff
@@ -5700,7 +5700,8 @@ private BErrorType createErrorType(BType lhsType, BType rhsType, BType detailTyp
 
     public BErrorType createErrorType(BType detailType, long flags, SymbolEnv env) {
```

Comment:
We call the `createErrorType` function from the `populateUndefinedErrorIntersection` function. In `populateUndefinedErrorIntersection`, we define a variable named `flag` and then set `public` to that variable. With this change, we don't need to do that.

Reply:
Make sense. Thanks!